docs/docs-content/palette-cli/commands/docs.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: "Docs"
+title: "Docs"
+description: "Learn how to enable the Palette documentation site in your local environment for an offline experience."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["palette-cli"]
+---
+
+The `docs` command starts the Palette documentation site in your local environment for an offline experience. A [container image](https://github.com/spectrocloud/librarium/pkgs/container/librarium) with the latest documentation content is downloaded and a local [Caddy](https://caddyserver.com/) server is started to serve the documentation site. You can access the Palette documentation site at `http://localhost:8080` using your browser.
+
+:::tip
+
+To stop the documentation site, type `Ctrl+C` in the terminal window where you started the documentation site.
+
+:::
+
+
+The `docs` command exposes the following flags.
+
+| Short Flag | Long Flag              | Description                                                              | Type    |
+|------------|------------------------|--------------------------------------------------------------------------|---------|
+| `-d`       | `--detach`             |  Start the container in the background and print the container ID.       | boolean  |
+| `-h`       | `--help`               |  Help with any command.                                                  | N/A     |
+|       | `--host`              |  The address of the host to expose the container on. The default value is set to `127.0.0.1`                                           | string  |
+|  | `--image` | The container image to use. The documentation image to use. Default value is `ghcr.io/spectrocloud/librarium:nightly`. | string |
+| | `--name` | The name of the container. Default value is `spectrocloud-docs`. | string |
+|        | `--port`               |  The port to expose the container on. The default value is set to `8080`. | string  |
+
+
+### Examples
+
+Start the documentation site in the background.
+
+```bash
+palette docs --detach
+```
+
+```shell hideClipboard
+{"status":"Pulling from spectrocloud/librarium","id":"nightly"}
+{"status":"Digest: sha256:754a75fe5b2724ad7fdd42389432fa8f81c2362880c236dabaa29ef6963b7ac6"}
```

Comment:
I wonder if we should remove the digest from these outputs? I'm thinking it will change and perhaps confuse people?

Reply:
It's example output, that's expected.